lib/node.js

```diff
@@ -118,6 +118,7 @@ declare class Buffer extends Uint8Array {
 
 declare module "buffer" {
   declare var kMaxLength: number;
+  declare export var Buffer: typeof Buffer;
```

Comment:
@mroch I tried the following:

```
const {Buffer} = require('buffer');

function readUInt8(buffer: Buffer, offset: number): number {
  return buffer.readUInt8(buffer, 'zero');
}
```

Which Flow (with this PR applied) correctly rejects with:

```
return buffer.readUInt8(buffer, 'zero');
^^^ string. This type is incompatible with the expected param type of
100:   writeUInt8(value: number, offset: number, noAssert?: boolean): number;
^^^^^^ number. See lib: flow/node.js:100
```